src/base/QXmppStun.cpp

```diff
@@ -1248,7 +1248,7 @@ void QXmppTurnAllocation::disconnectFromHost()
 
     // clear channels and any outstanding transactions
     m_channels.clear();
-    foreach (QXmppStunTransaction *transaction, m_transactions)
+    for (auto *transaction : m_transactions)
```

Comment:
Just interesting: why `qDeleteAll()` is not used here?

Reply:
Ok, as I see:
```
$ grep -r qDeleteAll * --exclude-dir=builddir/ | wc -l
0
```
And this is the answer.

Reply:
That's definitely the better way.